func_adl_xAOD/common/ast_to_cpp_translator.py

```diff
@@ -571,7 +598,7 @@ def visit_Call_Member(self, call_node):
         'Method call on an object'
 
         # Visit everything down a level.
-        self.generic_visit(call_node)
+        # self.generic_visit(call_node)
```

Comment:
@gordonwatts Can you verify if removing is correct? I feel it should be because we are calling `self.get_rep` on the `call_node.func.value` which will visit that node.

Reply:
And we also do it against the arguments, and as far as I know, those are the only two.

Let's try dropping it. Clearly, there are no tests that require it, so at some level it should be fine to remove it. If it fails, then we'll have a test to put in that will force this to exist.